CMakeLists.txt

```diff
@@ -221,9 +221,17 @@ else()
     message(VERBOSE "No custom malloc implementation")
 endif()
 
+# OpenSSL
+include(GetOpenSSL)
+get_openssl("${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_CURRENT_BINARY_DIR}")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/ssl")
+
+
 if (ENABLE_BZIP2 AND (ENABLE_LIBARCHIVE OR (ENABLE_ROCKSDB AND NOT WIN32)))
-    include(BundledBZip2)
-    use_bundled_bzip2(${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
+    include(GetBZip2)
+    get_bzip2(${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
+    # include(BundledBZip2)
+    # use_bundled_bzip2(${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
```

Comment:
these lines can be deleted
```suggestion
```

Reply:
Thanks. Yes you're right we can delete these commented lines. I will do it in the follow up commit.

Reply:
Heres the commit where I removed these comments: [816a882](https://github.com/apache/nifi-minifi-cpp/pull/1793/commits/816a882660102e6540eff98f4aaf203cd7ef217f)